qemu/tests/cfg/chardev_tls_encryption.cfg

```diff
@@ -0,0 +1,29 @@
+- chardev_tls_encryption:
+    type = chardev_tls_encryption
+    cert_dir = /etc/pki/qemu
+    start_vm = no
+    serials = ""
+    variants:
+        - host_to_guest:
+            gnutls_cmd_server = "cd %s &&"
```

Comment:
`cd ${cert_dir} &&`, same for the others

Reply:
Thanks and update.